apps/database/db/structure.sql

```diff
@@ -2,8 +2,8 @@
 -- PostgreSQL database dump
 --
 
--- Dumped from database version 13.10
--- Dumped by pg_dump version 14.2
+-- Dumped from database version 13.8
+-- Dumped by pg_dump version 13.10
```

Comment:
Please upgrade pg?

Reply:
I went to `pg_dump version 14.7 (Homebrew)`. I hope it's ok for you